subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/SpringBootPluginSmokeTest.groovy

```diff
@@ -46,6 +46,8 @@ class SpringBootPluginSmokeTest extends AbstractSmokeTest {
             }
         """.stripIndent()
 
+        withPluginValidation()
```

Comment:
Can we always enable this for smoke tests?

Reply:
We can't easily, because of the setup of the `plugins` block. It's different in every test and you have to know _when_ you can inject the validation check.